common/reviews/api/rush-lib.api.ts

```diff
@@ -149,6 +149,7 @@ class LockStepVersionPolicy extends VersionPolicy {
   ensure(project: IPackageJson): IPackageJson | undefined;
   readonly mainProject: string | undefined;
   readonly nextBump: BumpType;
+  update(newVersion: semver.SemVer): boolean;
```

Comment:
>semver.SemVer [](start = 21, length = 13)

Can we eliminate this dependency on the "semver" API contract?  That seems like a complex and unnecessary coupling